README.md

```diff
@@ -44,8 +44,10 @@ var feature = parser.Parse("Feature: ...");
 ```ruby
 # Ruby
 require 'gherkin3/parser'
+require 'gherkin3/token_scanner'
+
 parser = Gherkin3::Parser.new
-feature = parser.parse("Feature: ...")
+feature = parser.parse(Gherkin3::TokenScanner.new("Feature: ..."))
```

Comment:
To make the API more intuitive to use, we want `parse` to be able to take a scanner as well as a `String` or `IO`. See #95. Could you fix the code instead of the docs please? It should work the way the docs say.